examples/jsm/loaders/USDZLoader.js

```diff
@@ -675,24 +687,6 @@ class USDZLoader extends Loader {
 
 				}
 
-				if ( 'def Shader "diffuseColor_texture"' in data ) {
```

Comment:
These hardcoded shader evaluations were required since the actual shader had a different name than `PreviewSurface` and thus was not found.